src/openMVG/features/svgVisualization.cpp

```diff
@@ -15,7 +15,7 @@
 #include <algorithm> 
```

Comment:
@caymard can you check if we can remove the dependency from `#include <openMVG/localization/CCTagLocalizer.hpp>`?
I think it is just because some typedef that are declared there. See other comments below